recipes/horovod/meta.yaml

```
@@ -0,0 +1,105 @@
+{% set name = "horovod" %}
+{% set version = "0.28.1" %}
+{% set proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/horovod/horovod/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 88fbcd2815083607a5d27962f323eae5752e3eb96511aaac98258f280cac0f8e
+
+build:
+  number: 0
+  skip: true  # [win]
+  string: {{ proc_type }}_mpi_{{ mpi }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  ignore_run_exports_from:
+    - flatbuffers
+    - tensorflow
+    - pytorch
+  # Horovod supports multiple frameworks, and the package is built against TensorFlow and PyTorch.
+  # However, usually, the user only uses one of them simultaneously. (by importing either horovod.tensorflow or horovod.torch)
+  # Thus, TensorFlow and PyTorch are not listed in the `run` section, but in the `run_constrained` section.
+  # Users can manually install Horovod and only one of them.
+  # Both TensorFlow and PyTorch libraries are listed in `missing_dso_whitelist`, and a test is added to ensure Horovod libraries can be 
+  # correctly loaded when TensorFlow and PyTorch are installed.
+  # (horovod.tensorflow loads Horovod TensorFlow library and horovod.torch loads Horovod PyTorch library)
+  missing_dso_whitelist:
+    - "*/libtensorflow_framework.so.*"  # [linux]
+    - "*/libtensorflow_cc.so.*"  # [linux]
+    - "*/torch/lib/*.so"  # [linux]
+    - "*/libtensorflow_framework.*.dylib"  # [osx]
+    - "*/libtensorflow_cc.*.dylib"  # [osx]
+    - "*/_pywrap_tensorflow_internal.so"
+    - "*/torch/lib/*.dylib"  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != 'None']
+    - cmake >=3.13
+    - make  # [unix]
+    - python  # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+
```

You are using `flatc` directly in the build
```suggestion
   - flatbuffers
```